src/hw-2/index.ts

```diff
@@ -0,0 +1,75 @@
+import {getAutoSuggestUsers, getUserIndex} from './utils';
+import {User} from './types';
+import {validation} from './validation';
+import * as core from 'express-serve-static-core';
+
+const express = require('express');
+const app = express();
+const router: core.Router = express.Router();
```

Comment:
Ideally you have to keep routers separately with the place where server you define a server and run it. But for this small task let it be as it is.

Reply:
also, no much sense to define type of the variables in such place where type can be calculated by typescript automatically from the function return type.